sonar-project.properties

```diff
@@ -7,4 +7,6 @@ sonar.projectKey=alliander-opensource_short-term-forecasting
 
 # relative paths to source directories. More details and properties are described
 # in https://sonarcloud.io/documentation/project-administration/narrowing-the-focus/
-sonar.sources=.
\ No newline at end of file
+sonar.sources=openstf
+sonar.python.coverage.reportPath=/github/workspace/coverage.xml
```

Comment:
dit moet matchen met het `--cov-report`  argument. Wellicht is het zo dat als je daar `xml` meegeeft dat de default filename `coverage.xml`  is. Of dit path klopt weet ik ook niet, maar jij wel denk ik.

Reply:
Dit is een absoluut path hier, voor ander paden gebruik je relatieve paden is dat correct?

Reply:
yes, dat is inderdaad het geval